modules/dataquery/test/TestPlan.md

```diff
@@ -1,118 +1,202 @@
-16. Assert that: fields that were previously with all visits now are only with the selected visits in `Default Visits`.
-17. Make sure field visits are still updatable independently (line by line).
-18. Make sure if `Default Visits` are changed, it affects all fields.
-
-
-## Filters selection page
-
-1. Make sure no filter are already selected. The sentence `Currently querying for ALL candidates` should be displayed.
-2. Make sure the blue notification saying `Note that only candidates which you have permission to access in LORIS are included in results. Number of results may vary from other users running the same query.` is there.
-3. Make sure a preview of the number of candidates matched is displayed in the top-right hand corner.
-4. Make sure the `Add Condition` button triggers field select modal.
-   1. Add a field as a filter.
-   2. Assert that: the condition now appears in the filter list.
-   3. Assert that: `Add Condition` and `Import from CSV` buttons are now replaced by `Add "and" condition` and `Add "or" condition`.
-   4. Assert that: the top-right corner is updated.
-5. Remove added conditions with the trashbin icon on the right.
-6. Make sure `Import from CSV` button feature an upload modal.
-   1. Assert that: sending something different than CSV ends with an `Invalid CSV` alert.
-   2. Test file import by creating and importing different files with the preset options (candidates vs. sessions, DCCID vs. PSCID, with or without headers).
-7. Ensure conditions are organized by making several queries with various operators (AND/OR) and depths (condition groups).
-8. Click on `Run query` button.
+## Table of Contents
 
+1. [Setup](#setup)
+2. [Panels](#panels)
+3. [Recent Queries](#recent-queries)
+4. [Pin Tests](#pin-tests)
+5. [Fields Selection](#fields-selection)
+6. [Other parameters](#other-parameters)
+7. [Sync not Sync](#sync-not-sync)
+8. [Filters](#filters)
+9. [Run Query Page](#run-query-page)
 
-## Run query page
+## Setup
+
+Sign into your loris instance with an **admin** account.
+
+- select Admin Accounts and click the Add User button.
+- enter all required information.
+- select *ONE* site for example, "Rome".
+- select *ONE* project, for example Pumpernickel.
+- [x] Access Profile: View/Create Candidates and Timepoints - Own Sites.
+- [x] Data Dictionary: View Parameter Type Descriptions
+- [x] Data Query Tool: View/Download Cross-Modality Data
+
+## Panels
+
+- Ensure the module loads only for a user that has the `dataquery_view` permission. They must also have access to the dictionary module.
+
+- Assert that the following panels collapse and open
+
+   1. Instructions
+   2. Recent Queries
+   3. Next Steps (bottom-right corner)
+
+- Assert that `Continue to Define Fields` button in the main panel, and `Choose Fields` button in the `Next Steps` panel are redirecting to the same page.
+
+## Recent Queries
+
+### Perform the following assertions: (If no queries are available, add some)
```

Comment:
I will try something here, so bare with me. I had some difficulties reading the document, and I think it is linked to the document format. I think we should have a consistent difference between two types of lines in this document: 
1. `assertion/checks to do`, right now that is ok, they are listed.
2. `instructions-like lines`, such as "Repeat for `Shared Only`, `Named Only` and `No run times` checkboxes." or "Make a mix of the following actions" have varying formats.

So applying the markdown quote (it can also be multi-line) could do the trick for instruction like?
```suggestion
> Perform the following assertions: (If no queries are available, add some)
```